src/main/kotlin/InputUtils.kt

```diff
@@ -0,0 +1,71 @@
+private const val DELIMITER_1 = ","
+private const val DELIMITER_2 = ";"
+private const val ZERO_STRING = "0"
+private const val PREFIX_CUSTOM = "//"
+private const val SUFFIX_CUSTOM = "\\n"
+private const val ZERO_INDEX_WITHOUT_PREFIX = 3
+private const val ZERO_INDEX_OF_ONLY_INPUT = 5
+private const val INDEX_OF_CUSTOM = 2
+private const val ZERO = 0
+
+object InputUtils {
+    fun split(input: String): List<String> {
+        if (doesHaveCustom(input)) {
+            return splitByCustom(input)
+        }
+
+        if (input.isBlank()) {
+            return listOf(ZERO_STRING)
+        }
+
+        return splitByDefault(input)
+    }
+
+    fun convertToNumber(splitInput: List<String>): List<Int> {
+        try {
+            return parse(splitInput)
+        } catch (e: NumberFormatException) {
+            throw RuntimeException("숫자만 허용됩니다.")
+        }
+    }
+
+    fun getSum(numbers: List<Int>): Int {
+        return numbers.sum()
+    }
+
+    private fun splitByDefault(input: String): List<String> {
+        return input.split(DELIMITER_1)
+            .flatMap { it.split(DELIMITER_2) }
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        return input.split(DELIMITER_1, DELIMITER_2)
```